tools/generators/migrate-fixed-versions/index.spec.ts

```diff
@@ -72,6 +98,26 @@ describe('migrate-fixed-versions generator', () => {
     `);
   });
 
+  it('should not migrate third party dev dependencies', async () => {
+    tree = setupDummyPackage(tree, {
+      name: '@proj/react-positioning',
+      version: '9.0.0-alpha.0',
+      dependencies: {},
+      devDependencies: {
+        'popperjs/core': '^1.0.0',
```

Comment:
```suggestion
        '@popperjs/core': '^1.0.0',
        'tslib': '^2.0.0',
```
Can we use real deps there?